workflow/scripts/extract-fp-fn.py

```
@@ -60,6 +74,20 @@
                     #No VAF information available -> float na
                     vaf = float('nan')
 
+            elif c.cls == Class.TP_query and snakemake.wildcards.classification == "tp":
```

Ruff (0.15.20)

[error] Undefined name `snakemake`

(F821)